experiments/compositions/composition/api/v1alpha1/expanderversion_types.go

```diff
@@ -53,6 +59,9 @@ type ExpanderVersionSpec struct {
 	// +kubebuilder:validation:Enum=job;grpc
 	// +kubebuilder:default=job
 	Type ExpanderType `json:"type"`
+
+	// ExpanderConfig GVK
+	Config ExpanderConfigGVK `json:"config,omitempty"`
```

Comment:
Don't we need a name and namespace to point at an instance of the configuration?

Reply:
NM missed this was metadata not individual expander config.